core/scripts/server.js

```diff
@@ -246,7 +246,11 @@ app.get('*', (req, res, next) => {
   }
 
   if (config.server.dynamicConfigReload) {
-    delete require.cache[require.resolve('config')]
+    const cachedConfigModule = require.cache[require.resolve('config')]
+    if (cachedConfigModule) {
+      delete cachedConfigModule.parent.children
+      delete cachedConfigModule
```

Comment:
Variables should not be deleted, only properties of object, i'd suggest:
```suggestion
      delete require.cache[require.resolve('config')]
```